### ISSUE TITLE
`repeat.for="player of players"` Syntax is tripping me up constantly

### ISSUE THREAD

**Issue by no1melman** [aurelia/framework#532]
So, I don't think this will get much "traction" but the syntax:

```
repeat.for="player of players"
```

Is really tripping me up. Reason being, if you use C# for example:

```csharp
foreach(var player in players)
```

If you read that out loud in plain english it makes sense. `for each player in players`. However, the aurelia syntax just doesn't make any sense.

I could easily live with

```
repeat.for="player in players"
```

Which makes far more sense. It's more readable, more memorable.

I know where this syntax came from, the ES6 spec in `for...of` to iterate through collections, but to use the JavaScript language as a starting point for the Aurelia stuff isn't probably a good start seeing how frustrating people find JavaScript to begin with....

Is this tripping anyone else up frequently?? I know it may be a "big" breaking change, but nothing a simple `Ctrl+Shift+H` in VS couldn't solve or a `Ctrl+H` in Notepad++ couldn't solve....

**Comment by EisenbergEffect** [comment]
The syntax will remain based on the ES 2015 for..of loop. That's the standard for the platform and it's important for us to stick to it. As more developers learn and get comfortable with modern JavaScript, it should become more natural for them.
